api/models/GameStateRow.js

```diff
@@ -72,7 +73,7 @@ module.exports = {
     phase: {
       type: 'number',
       required: true,
-      isIn: [ 1, 2, 3, 4, 5, 7 ],
+      isIn: Object.values(GamePhase),
```

Comment:
Same idea here; validate via direct comparison to the `GamePhase` object for centralized source of truth on valid phases